src/internal/m365/collection/exchange/collection.go

```diff
@@ -194,13 +194,17 @@ func (col *prefetchCollection) streamItems(
 		colProgress chan<- struct{}
 
 		user = col.user
-		log  = logger.Ctx(ctx).With(
-			"service", path.ExchangeService.String(),
-			"category", col.Category().String())
 	)
 
+	ctx = clues.Add(
+		ctx,
+		"item_stream_category", col.Category().String())
```

Comment:
nit: `category` is a well known label, should we keep it? Also add back the `service`?

Reply:
Hmm, sure, I suppose we can.  The worry was that we're clobbering the category value established at a higher scope, but that probably isn't much of a worry.